fdbserver/tester.actor.cpp

```diff
@@ -1766,7 +1766,7 @@ ACTOR Future<Void> runTests(Reference<AsyncVar<Optional<struct ClusterController
 	}
 
 	// Read cluster configuration
-	if (useDB) {
+	if (useDB && g_network->isSimulated()) {
```

Comment:
I was meaning to do this in a. different pr but can you also replace the code below:
```
state DatabaseConfiguration configuration;
		loop {
			try {
				tr.setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
				tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
				tr.setOption(FDBTransactionOptions::READ_LOCK_AWARE);
				RangeResult results = wait(tr.getRange(configKeys, CLIENT_KNOBS->TOO_MANY));
				ASSERT(!results.more && results.size() < CLIENT_KNOBS->TOO_MANY);
				configuration.fromKeyValues((VectorRef<KeyValueRef>)results);
				break;
			} catch (Error& e) {
				wait(tr.onError(e));
			}
		}
```
with `getDatabaseConfiguration(cx)`?